.github/workflows/android.yml

```diff
@@ -45,3 +46,61 @@ jobs:
         PYTHON_EXECUTABLE=python bash .ci/scripts/setup-linux.sh "${BUILD_TOOL}"
         # Build Android demo app
         bash build/test_android_ci.sh
+
+        mkdir -p artifacts-to-be-uploaded
+        # Copy the app and its test suite to S3
+        cp examples/demo-apps/android/LlamaDemo/app/build/outputs/apk/debug/*.apk artifacts-to-be-uploaded/
+        cp examples/demo-apps/android/LlamaDemo/app/build/outputs/apk/androidTest/debug/*.apk artifacts-to-be-uploaded/
```

Comment:
(Not a blocker for this PR)

Can we upload these as well?
cmake-android-out/lib/libcpuinfo.a
cmake-android-out/lib/libpthreadpool.a
cmake-android-out/lib/libportable_kernels.a
cmake-android-out/lib/libportable_ops_lib.a
cmake-android-out/lib/libexecutorch.a
cmake-android-out/lib/libexecutorch_no_prim_ops.a
cmake-android-out/lib/libextension_data_loader.a
cmake-android-out/lib/libextension_module.a
cmake-android-out/lib/libXNNPACK.a
cmake-android-out/lib/libxnnpack_backend.a

Reply:
(Or, maybe we can build them in a separate workflow)

Reply:
Sounds good, I can easily add them right away to make them available on gha-artifacts bucket, we can decide if they need to be uploaded elsewhere later.